File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
     
        int max_wealth = INT_MIN;
        
        for(int i = 0; i < accounts.size(); i++) {
            int wealth = 0;
            for(int j = 0; j < accounts[0].size(); j++) {
                wealth += accounts[i][j];
            }
            if(wealth > max_wealth) {
                max_wealth = wealth;
            }
        }
        
        return max_wealth;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        
        rows, columns = len(accounts), len(accounts[0])
        max_wealth = -1
        for i in range(rows):
            current_wealth = 0
            for j in range(columns):
                current_wealth += accounts[i][j]
            if(current_wealth > max_wealth):
                max_wealth = current_wealth
        return max_wealth